hazelcast-client/src/test/java/com/hazelcast/client/cache/ClientCachingProviderTest.java

```diff
@@ -62,8 +65,6 @@ public void setup() {
             throw new AssertionError("Could not construct named hazelcast client instance: " +
                     e.getMessage());
         }
-        instances.add(instance1);
```

Comment:
`instance1` and `instance2` have already been added to `instances` in `createHazelcastInstance` method.